GitHub-Test-4-PyNLPDiscuss/rorange.md

```diff
@@ -0,0 +1,23 @@
+#Remi's Git Test 4 Submission
+
+1.) I found the *Entity Ruler* interesting because it doesn't branch too far away from what we're discussing and allows us to take a little more control over the code we're working with. It feels like bending the code to do what I want it to do.
```

Comment:
I agree with you mentioning the idea of "bending the code" to be able to do what you want. Entity Ruler is a very neat method and would seemingly be a good starting point for somebody wanting to learn.